java/code/src/com/redhat/rhn/frontend/action/errata/CreateAction.java

```diff
@@ -134,6 +148,7 @@ public ActionForward execute(ActionMapping mapping,
         e.setOrg(user.getOrg());
 
         ErrataManager.storeErrata(e);
+        ErrataManager.publish(e, channelAction.getChannelIdsFromRhnSet(set), user);
```

Comment:
is the whole concept of "publishing an errata" still valid? can it be an errata in another state that is "not published"?
shouldn't we do this as part of the creation of the errata?

Reply:
The concept of publishing an errata per se remains - it's just that before, one could publish both an unpublished or a published errata - now unpublished errata cease to exist. So the only remaining option is to re-publish something that was already published before - which is a valid thing to do.

On a high level, I completely agree the "first publishing" should be part of errata creation. Whether that means to wrap those methods together in this specific case is something I would leave for the follow up PR.

So see you there, if you agree.

Reply:
It would be nice to address this in a follow up PR for clarify. Thanks